docs/api/declarations.rst

```diff
@@ -784,17 +784,17 @@ Exmples for :meth:`Declaration.__add__`:
    ['IRoot1']
    >>> [iface.getName() for iface in spec2]
    []
-   >>> spec2 += Declaration(IRoot2, IDerived2)
+   >>> spec2 += Declaration(IDerived2)
```

Comment:
I think it would also work to just reverse the order, e.g., `Declaration(IDerived2, IRoot2)`. If that works, semantically that should mean the same thing, and would preserve the value of this example (showing that `IRoot2` remains at the back — this was part of #238).

There was a similar case above, I think.